src/ludamus/templates/components/navbar.html

```
@@ -19,7 +18,7 @@
                                  height="24"
                                  class="h-6 w-auto rounded object-contain">
                         {% endif %}
-                        {{ current_sphere.name }}
+                        <span {% if current_sphere.logo_url or current_sphere.enabled_pages|length > 1 %} class="max-sm:hidden"{% endif %}>{{ current_sphere.name }}</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)